helm/charts/determined/templates/NOTES.txt

```diff
@@ -0,0 +1,9 @@
+Thank you for installing {{ .Chart.Name }}. Your release is named: {{ .Release.Name }}.
+
+It may take several minutes for your deployment to start up. 
+Once it is up you can get the IP address of your deployment by running:
+
+kubectl get service determined-master-service-{{ .Release.Name }} -n {{ .Release.Namespace}} --output jsonpath='{.status.loadBalancer.ingress[0].ip}'
+
+Once you have the IP address set the master address with `export DET_MASTER=<ip address>`.
```

Comment:
non-blocking: It would be great if we could print the `export` command with the address of the master filled in. I'd also print it on a new line so it's easier to copy.

Reply:
We can't do that here since Helm exits out before all resources are ready (unless users use `--wait`). We do include a command for them to look up to get the ip address.